sdk/identity/azure-identity-v2/src/main/java/com/azure/identity/v2/AuthorizationCodeCredential.java

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.v2;
+
+import com.azure.identity.v2.implementation.client.ConfidentialClient;
```

Comment:
Also, move [module-info.java](https://github.com/Azure/azure-sdk-for-java/blob/8803ca5ec7e2aa2fb04659b1801fb929cd26c472/sdk/identity/azure-identity-v2/src/main/module-info.java) to `src/main/java` directory and update the module name to `com.azure.v2.identity`.